### PR TITLE
Remove wrong parentheses in two UnicodeError messages

### PR DESCRIPTION
**Pull request by nalimilan**

### CODE REVIEW

File: base/unicode/UnicodeError.jl [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 ##    Error messages for Unicode / UTF support
 
-const UTF_ERR_SHORT             = "invalid UTF-8 sequence starting at index <<1>> (0x<<2>>) missing one or more continuation bytes)"
-const UTF_ERR_CONT              = "invalid UTF-8 sequence starting at index <<1>> (0x<<2>> is not a continuation byte)"
+const UTF_ERR_SHORT             = "invalid UTF-8 sequence starting at index <<1>> (0x<<2>>) missing one or more continuation bytes"
+const UTF_ERR_CONT              = "invalid UTF-8 sequence starting at index <<1>> (0x<<2>> is not a continuation byte"
```
Comment (tkelman) [inline base/unicode/UnicodeError.jl]: is this missing a close paren after the `<<2>>` ?

Comment (nalimilan) [inline base/unicode/UnicodeError.jl]: Damn, removing one mistake and adding another one in the opposite direction...

**Comment by nalimilan** [comment]
Actually, I hadn't correctly understood the error message. Removing the first closing parenthesis makes more sense.

**Comment by tkelman** [comment]
I disagree (and could you wait for review?), the first parenthesis is just indicating what is present at the given index

**Comment by tkelman** [comment]
there's another in `UTF_ERR_NOT_SURROGATE`

**Comment by nalimilan** [comment]
Sorry, I thought that was trivial enough once you had spotted my original mistake... I think the solution I merged makes sense: a continuation byte would be expected after `0x<<2>>`. But maybe yours is better. Good catch for the other missing parenthesis.

See https://github.com/JuliaLang/julia/pull/14554

**Pull request merged**